clusters/app.ci/release-reimport-controller/deployment.yaml

```diff
@@ -29,6 +29,7 @@ spec:
       - command:
         - /usr/bin/release-reimport-controller
         - start
+        - --dry-run
```

Comment:
You'll need to rebase against the latest/greatest changes.  This parameter was removed recently.

Reply:
I literally just generated that an hour ago... talk about unfortunate timing, ha ha ha

Reply:
This isn't your fault actually...  It needs to be corrected in the correct place...
https://github.com/openshift/release/pull/49343
That will need to merge and then this file should drop out after you rebase and do a:
`make release-controllers`

Reply:
Sorry for the churn!